src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -717,8 +728,20 @@ private void initializeInputPreloadExecutorService() {
     }
 
     private Void logMessageOnBatchCompletion(final BatchCompletionCallbackFunctionArgument arg) {
-        progressMeter.update(null);
         logger.info("Done importing batch " + arg.batchCount + "/" + arg.totalBatchCount);
+        logger.info("List of samples imported in batch " + arg.batchCount + ":");
+        int index = 0;
+        final int sampleCount = sampleNameToVcfPath.size();
+        final int updatedBatchSize = (batchSize == DEFAULT_ZERO_BATCH_SIZE) ? sampleCount : batchSize;
+        final int startBatch = (arg.batchCount - 1) * updatedBatchSize;
+        final int stopBatch = arg.batchCount * updatedBatchSize;
```

Comment:
It seems like these calculations could be moved into the batch argument class itself but maybe that's not worth it since they're only used here.

Reply:
Left these since they're only used here...